index.js

```diff
@@ -18,27 +15,27 @@ const PORT = process.env.PORT || 4000;
 
 const app = express();
 
-
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(passport.initialize());
 app.use(passport.session());
 app.use(validator());
 
-app.get('/', (req, res) => res.status(200).send({
-  message: 'Welcome to the Author\'s haven',
-}));
-
 app.use('/api/v1/auth', auth);
 app.use('/api/v1/auth_twitter', twitterRouter);
 app.use('/api/v1/profiles', profiles);
 app.use('/api/v1/forgotpassword', user);
 app.use('/api/v1/users', user);
 passportAuth();
 
-app.use((req, res) => res.status(404).json({ message: 'not found' }));
+// Default to here when an invalid endpoint is entered
+app.use('/', (req, res) => res.status(200).json({
+  success: true,
+  message: 'Welcome to Authors Haven by Heimdal'
+}));
 
+// app.use((req, res) => res.status(404).json({ message: 'not found' }));
```

Comment:
Why is this commented out?

Reply:
It's commented out because there was a duplicate and the app.use('/', (req, res))... above it sort of catches all the request so it wouldn't be called.

Reply:
How do you handle 404?

Reply:
`/` is not a catch all. `*` is a catch all

Reply:
Okay thanks, I will fix this now.